<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendTrajectory</class>
 <widget class="QDialog" name="sendTrajectory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>341</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>377</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AileronPos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="aileron0"/>
     </item>
     <item row="2" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle1"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ElevatorPos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="2">
      <widget class="QLineEdit" name="time2"/>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QLineEdit" name="time1"/>
     </item>
     <item row="4" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator2"/>
     </item>
     <item row="5" column="0" rowspan="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" rowspan="2">
      <widget class="QLineEdit" name="time3"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="time0">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="inputMask">
        <string comment="0" extracomment="0"/>
       </property>
       <property name="text">
        <string comment="0" extracomment="0"/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ThrottlePos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator1"/>
     </item>
     <item row="4" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron2"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="elevator0"/>
     </item>
     <item row="2" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron1"/>
     </item>
     <item row="4" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="throttle0"/>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Index</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" rowspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0" rowspan="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" rowspan="2">
      <widget class="QLineEdit" name="time4"/>
     </item>
     <item row="14" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle7"/>
     </item>
     <item row="14" column="1" rowspan="2">
      <widget class="QLineEdit" name="time7"/>
     </item>
     <item row="16" column="1" rowspan="2">
      <widget class="QLineEdit" name="time8"/>
     </item>
     <item row="14" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron7"/>
     </item>
     <item row="10" column="1" rowspan="2">
      <widget class="QLineEdit" name="time5"/>
     </item>
     <item row="8" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator4"/>
     </item>
     <item row="6" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle3"/>
     </item>
     <item row="6" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron3"/>
     </item>
     <item row="6" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator3"/>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1" rowspan="2">
      <widget class="QLineEdit" name="time9"/>
     </item>
     <item row="16" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator8"/>
     </item>
     <item row="18" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator9"/>
     </item>
     <item row="10" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron5"/>
     </item>
     <item row="8" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle4"/>
     </item>
     <item row="10" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle5"/>
     </item>
     <item row="12" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator6"/>
     </item>
     <item row="11" column="0" rowspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle6"/>
     </item>
     <item row="13" column="0" rowspan="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" rowspan="2">
      <widget class="QLineEdit" name="time6"/>
     </item>
     <item row="12" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron6"/>
     </item>
     <item row="14" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator7"/>
     </item>
     <item row="15" column="0" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="16" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron8"/>
     </item>
     <item row="16" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle8"/>
     </item>
     <item row="8" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron4"/>
     </item>
     <item row="9" column="0" rowspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" rowspan="2">
      <widget class="QLineEdit" name="elevator5"/>
     </item>
     <item row="20" column="0" colspan="5">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Send Trajectory</string>
       </property>
      </widget>
     </item>
     <item row="18" column="4" rowspan="2">
      <widget class="QLineEdit" name="throttle9"/>
     </item>
     <item row="18" column="3" rowspan="2">
      <widget class="QLineEdit" name="aileron9"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
